linalgo/annotate/serializers.py

```diff
@@ -81,7 +81,7 @@ def _serialize(instance):
             'body': instance.body or '',
             'annotator_id': annotator_id,
             'document_id': instance.document.id,
-            'created': instance.created.strftime('%Y/%m/%d %H:%M:%S.%f'),
+            'created': instance.created.isoformat(),
```

Comment:
This way it works with 
"""
elif isinstance(created, str):
            created = datetime.fromisoformat(created)
"""
(see lines 177, 178  in linalgo/annotate/models.py)

Reply:
Noted, thanks. I need to check that it works with Linhub's backend deserialization as well. It should be OK.